### PR TITLE
Add a message for experimental builds

### PR DESCRIPTION
**Pull request by ImMorpheus**

### CODE REVIEW

**Comment by liach** [comment]
are you sure most 8.0 users are not developers?

**Comment by ImMorpheus** [comment]
https://github.com/SpongePowered/SpongeCommon/issues?utf8=%E2%9C%93&q=8.0.0

**Comment by gabizou** [comment]
@liach it's more of "8.0 is slated to change both API wise and implementation wise, so reporting more gameplay bug issues with testing on bleeding isn't useful to the development team's time."

File: message/issue/experimental.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hey @${author}! You seem to be running an experimental version (${version}) of ${repository_name} that should not be run on production systems.
+
+Plugin compatibility WILL break often on experimental systems, so unless you are a plugin developer assisting the next version of the API, please retest on API (${stable}).
```
Comment (kashike) [inline message/issue/experimental.md]: `WILL` -> `**will**`

Comment (kashike) [reply message/issue/experimental.md]: `please retest on API (${stable})` sounds weird.

**Review by kashike — CHANGES_REQUESTED** [review]

File: message/issue/experimental.md (hunk 1 above)
Comment (ImMorpheus) [reply message/issue/experimental.md]: ```Please re-test on the reccomended build or latest build of Sponge (${stable}). ``` ?

(credit to aaron)

Comment (liach) [reply message/issue/experimental.md]: `retest` -> `reproduce`

**Review by liach — APPROVED** [review]

File: message/issue/experimental.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Hey @${author}! You seem to be running an experimental version (${version}) of ${repository_name} that should not be run on production systems.
+
+Plugin compatibility **will** break often on experimental systems, so unless you are a plugin developer assisting the next version of the API, please reproduce on the reccomended build or latest build of Sponge (${stable}).
```
Comment (liach) [inline message/issue/experimental.md]: Eh, recommended is a typo here

File: message/issue/experimental.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+This was an automated post, if you strongly disagree with this bot's response please reply stating why.
```
Comment (liach) [inline message/issue/experimental.md]: Your tone should be slightly weaker, e.g. `This was an automated post. If you have a reason to run on ${version}, please clarify.`

**Review by liach — CHANGES_REQUESTED** [review]

File: message/issue/experimental.md (hunk 3 above)
Comment (ImMorpheus) [reply message/issue/experimental.md]: tbh I was thinking of removing this part for consistency. After all, none of the bot messages has the "this is an automated post.." disclaimer.

File: message/issue/experimental.md [hunk 4]
```diff
@@ -1,5 +1,3 @@
 Hey @${author}! You seem to be running an experimental version (${version}) of ${repository_name} that should not be run on production systems.
 
-Plugin compatibility **will** break often on experimental systems, so unless you are a plugin developer assisting the next version of the API, please reproduce on the reccomended build or latest build of Sponge (${stable}).
-
-This was an automated post, if you strongly disagree with this bot's response please reply stating why.
\ No newline at end of file
+Plugin compatibility **will** break often on experimental systems, so unless you are a plugin developer assisting the next version of the API, please reproduce on the recommended build or latest build of Sponge (${stable}).
```
Comment (liach) [inline message/issue/experimental.md]: Replace last `Sponge` with `${repository_name} `

**Review by liach — APPROVED** [review]

File: include/github/apply.xml [hunk 5]
```diff
@@ -125,6 +125,12 @@
                   <label>
                     <add>branch: bleeding</add>
                   </label>
+                  <comment src="issue/experimental.md">
+                    <tokens>
+                      <token name="version" value="8.0.0-BETA"/>
+                      <token name="stable" value="7"/>
```
Comment (kashike) [inline include/github/apply.xml]: Be explicit: `7.0.0`.

File: message/issue/experimental.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+Hey @${author}! You seem to be running an experimental version (${version}) of ${repository_name} that should not be run on production systems.
```
Comment (kashike) [inline message/issue/experimental.md:1]: `should not be run` vs `should not be ran`?

**Review by kashike — CHANGES_REQUESTED** [review]

File: message/issue/experimental.md (hunk 6 above)
Comment (liach) [reply message/issue/experimental.md:1]: `run`-`ran`-`run`. The past participle of `run` is `run`, so the original is right.

File: include/github/apply.xml (hunk 5 above)
Comment (liach) [reply include/github/apply.xml]: It may apply to 7.1 that will be released soon.

Comment (ImMorpheus) [reply include/github/apply.xml]: `7.x` ?

Comment (kashike) [reply include/github/apply.xml]: `7.x` works for me.

**Pull request closed without merging**